modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,154 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* The `git`, `jq`, and `wget` tools are available.
```

Comment:
`available` seems both redundant and vague. Do these tools need to be installed or included in a container image - where does the user find them - in a workspace?

Reply:
That's for consistency for the rest of the documentation, where we frequently use "The `foo` tool is available".

Reply:
:) I've never seen a list of cooking ingredients using the convention of **`5 eggs` are available**.
It may be **5 boiled `eggs`** though.

Reply:
OK rewritten.